packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -231,7 +231,7 @@ export class CrossChainMessenger {
    *
    * @returns The L2OutputOracle parameters.
    */
-  public async getL2OutputOracleParameters(): Promise<L2OutputOracleParameters> {
+  public async getL2OutputAfterOracleParameters(): Promise<L2OutputOracleParameters> {
```

Comment:
Lawl I think this is an errant find/replace

Reply:
barf